### DISCUSSION TITLE
Pan V1 and V3 assistance

### DISCUSSION THREAD

**Discussion by pdobrien3** [gtxaspec/wz_mini_hacks#482] [Q&A]
I have a V3 and a Pan V1 both with wz_mini installed and using rtsp streams (both hi rez and low rez).  I am using Frigate to record streams and do object detection.  The V3 work pretty flawless outside of a little pixilation here and there. My settings are exactly the same for both cameras. I can see both streams on both cameras in frigate and can see object detection clips on both cameras in frigate. The problem comes when i try to view recoded video clips in frigate. the V3 plays fine, the Pan V1 wont play the clips. I started to play and noticed that if i download the V3 files from frigate, they will play in just about anything, including windows media player.  The Pan V1 will only play in VLC.  I am stuck on where to turn to further diagnosis this issue. 

here are the ffmpeg probes for both cameras from frigate:  

V3
[ { "return_code": 0, "stderr": {}, "stdout": { "programs": [], "streams": [ { "avg_frame_rate": "20/1", "codec_long_name": "H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10", "height": 360, "width": 640 } ] } }, 

{ "return_code": 0, "stderr": {}, "stdout": { "programs": [], "streams": [ { "avg_frame_rate": "20/1", "codec_long_name": "H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10", "height": 1080, "width": 1920 }, { "avg_frame_rate": "0/0", "bit_rate": "256000", "codec_long_name": "PCM signed 16-bit big-endian" } ] } } ]

Pan V1
[ { "return_code": 0, "stderr": {}, "stdout": { "programs": [], "streams": [ { "avg_frame_rate": "5/1", "codec_long_name": "H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10", "height": 360, "width": 640 } ] } }, 

{ "return_code": 0, "stderr": {}, "stdout": { "programs": [], "streams": [ { "avg_frame_rate": "15/1", "codec_long_name": "H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10", "height": 1080, "width": 1920 }, { "avg_frame_rate": "0/0", "bit_rate": "128000", "codec_long_name": "PCM signed 16-bit big-endian" } ] } } ]

The V3 is running 4.36.9.139 and the Pan V1 is running 4.10.8.1002.  I am under the impression that 4.10.8.1002 is the last version before Wyze started to work against wz_mini. 

any assistance would be greatly appreciated. thanks.

**Comment by bjs-pdx** [comment]
I don't have any input to your question but your post is relevant to something of mine.  You said "the Pan V1 is running 4.10.8.1002. "  What network adapter are you using or are you accessing the cameras over wifi?

**Comment by pdobrien3** [comment]
WiFi

**Comment by pdobrien3** [comment]
Just an update on where I am with the Pan V1. I have recordings working without audio. I also have been unable to get the `ENABLE_SELFHOSTED_MODE="true"` to work. Even though it is enabled, I can still see video in the app. Here are my settings;

```
RTSP_HI_RES_ENABLED="true"
RTSP_HI_RES_ENABLE_AUDIO="false"
RTSP_HI_RES_MAX_BITRATE="2048"
RTSP_HI_RES_TARGET_BITRATE="1024"
RTSP_HI_RES_ENC_PARAMETER="4"
RTSP_HI_RES_FPS="15"

RTSP_LOW_RES_ENABLED="true"
RTSP_LOW_RES_ENABLE_AUDIO="false"
RTSP_LOW_RES_MAX_BITRATE="2048"
RTSP_LOW_RES_TARGET_BITRATE="1024"
RTSP_LOW_RES_ENC_PARAMETER="4"
RTSP_LOW_RES_FPS="5"
```

The encoding parameter setting of 4 is what made everything start working.

relevant frigate config:

 ```
 Pan_V1_camera: # <------ Name the camera
    ffmpeg:
      hwaccel_args: preset-vaapi
      input_args: preset-rtsp-udp
      output_args: 
        record: preset-record-generic
        detect: -dn -an -f rawvideo -pix_fmt yuv420p
```
